packages/caml-mode/caml-mode.4.02/opam

```diff
@@ -20,6 +20,12 @@ remove: [
   [make "uninstall-ocamltags" "BINDIR=%{bin}%"]
 ]
 depends: []
+depexts: [
+  [["debian"] ["emacs-nox"]]
+  [["ubuntu"] ["emacs-nox"]]
+  [["macosx"] ["emacs"]]
```

Comment:
This `macosx` line should definitely go -- the only interpretation that's valid right now is `[["osx"]["homebrew"]]` or `[["osx"]["macports"]]` and its not needed anyway due to Emacs being included by default in OSX.